src/screens/LocationDetailsScreen.tsx

```diff
@@ -80,6 +84,32 @@ export const LocationDetailsScreen = () => {
     }, 1000);
   }, []);
 
+  useEffect(
+    () =>
+      navigation.addListener("beforeRemove", (e) => {
+        const action = e.data.action;
+        if (newWaitTime === 0) {
+          return;
+        }
```

Comment:
Do we want to handle the edge case where they might put an estimate of 0 mins. Like there is no line?